manual/manual/cmds/unified-options.etex

```diff
@@ -308,6 +308,13 @@ modules contained in the library.  When compiling a module (option
 always be linked if it is put in a library and this library is linked.
 }%notop
 
+\nat{%
+\item["-linscan"]
+Use linear scan register allocation. Compilation time with this allocator is
+typically shorter than with the usual graph coloring allocator. On the other
+hand, the generated code can have longer running times.
```

Comment:
This wording sounds a bit too careful.  I'd say something like "Compiling with this allocator is faster than with the usual graph coloring allocator, sometimes quite drastically for long functions and modules.  The generated code can be a bit slower."